components/feature/script/README.md

```diff
@@ -0,0 +1,31 @@
+# [Android Components](../../../README.md) > Feature > Script
+
+Feature that enables listening and responding to slowScript callback.
+
+## Usage
+
+### Setting up the dependency
+
+Use Gradle to download the library from [maven.mozilla.org](https://maven.mozilla.org/) ([Setup repository](../../../README.md#maven-repository)):
+
+```Groovy
+implementation "org.mozilla.components:feature-script:{latest-version}"
+```
+
+###Integration
+
+Initialize the feature:
+
+```kotlin
+val slowLoadingFeature = SlowLoadingFeature(
+    appContext,
+    sessionId,
+    store
+),
```

Comment:
We have two required parameters in our feature, so let's document how we believe apps should use it. Here are some examples of docs you could possibly add:

```suggestion
val slowLoadingFeature = SlowLoadingFeature(
    sessionId,
    store,
    ::handleShowNotification,
    ::handleDismissNotification,
)

fun handleShowNotification(notification: SlowScriptCallback) {
  // Prompt the user to..
  // stop the slow-running script from continuing:
  notification.stopSlowScript()
  // allow the script to continue a little longer:
  notification.continueSlowScript()
}

fun handleDismissNotification() {
  // Remove any prompt for the user.
}
```

Reply:
Here are some examples of docs we try to write if you're looking for examples: https://github.com/mozilla-mobile/android-components/tree/main/components/feature/prompts